core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -64,6 +64,9 @@ oppia.directive('answerFeedbackPair', [
                   $scope.data.learnerAnswer, interaction.id,
                   interaction.customizationArgs));
             }
+            if ($scope.data.learnerAnswer === 'I would like a hint.') {
+              return $scope.data.learnerAnswer;
+            }
```

Comment:
I'm a little concerned about the modelling here. Shouldn't we have a boolean field in the data object (or somewhere relevant) stating whether a hint was requested, and doing ng-if="wasHintRequested" in whatever div contains the short answer? Otherwise you run into issues with translation of platform text, etc.